mocked-api/schemasData.ts

```diff
@@ -785,4 +793,80 @@ export const schemasData: { [key: string]: object } = {
       },
     },
   },
+  "azure_eventhubs_source_0.1": {
+    type: "object",
+    additionalProperties: false,
+    required: [
+      "azure_namespace_name",
+      "azure_eventhub_name",
+      "azure_shared_access_name",
+      "azure_shared_access_key",
+      "azure_blob_account_name",
+      "azure_blob_access_key",
+      "azure_blob_container_name",
+    ],
+    properties: {
+      azure_namespace_name: {
+        title: "Eventhubs Namespace",
+        description: "The Event Hubs namespace.",
+        type: "string",
+      },
+      azure_eventhub_name: {
+        title: "Eventhubs Name",
+        description: "The Event Hub name.",
+        type: "string",
+      },
+      azure_shared_access_name: {
+        title: "Share Access Name",
+        description: "The Event Hubs SAS key name.",
+        type: "string",
+        "x-group": "credentials",
+      },
+      azure_shared_access_key: {
+        title: "Share Access Key",
+        "x-group": "credentials",
+        oneOf: [
+          {
+            title: "Share Access Key",
+            description: "The key for the EventHubs SAS key name.",
+            type: "string",
+            format: "password",
```

Comment:
@kelvah probably this is not a problem of this issue but the value of this property is visible in Process Details. Is it a know issue?
![image](https://user-images.githubusercontent.com/1536609/183028449-69c06fa9-7064-4cf2-aa6c-659dd04d22b1.png)

Reply:
This will be covered by #101.